ltr/concourse/deploy.py

```diff
@@ -10,11 +10,17 @@
 role = os.environ["ROLE_ARN"]
 model_name = os.environ["SCRIPT_INPUT_DATA"].strip()
 
+try:
+    model_tag = os.environ["MODEL_TAG"]
+except KeyError:
+    model_tag = ""
```

Comment:
This can be `os.getenv("MODEL_TAG", "")`

Reply:
Even better, you can just use `os.getenv("MODEL_TAG") here, and then change line 49 to `model_location = model_tag if model_tag else model_name` - `None` is falsey.